pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -332,3 +332,12 @@ def predict_step(self, *args, **kwargs):
     def post_training_step(self):
         if not self.lightning_module.automatic_optimization:
             self.model.require_backward_grad_sync = True
+
+    @classmethod
```

Comment:
I think we should split this to a separate PR to ensure this PR does only one thing